kustomization/base/api-gateway-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: api-gateway-service
+  namespace: my-app-namespace
+spec:
+  type: NodePort

```

Comment:
Any reason this is of NodePort type?

Reply:
Wanted to check it on the internet at that time.

Reply:
Updated from NodePort to ClusterIP.